run_nmos_testing.ps1

```diff
@@ -0,0 +1,66 @@
+# Windows script to start nmos-testing tool
+
+param(
+    [switch]$e,  # Default is $false
+    [switch]$u,  # Default is $false
+    [string]$c = "../../../BuzzVirtual-Gen2-documentation/certificates/"  # Default certificate path
```

Comment:
Can we use REPO_ROOT approach as in the original? CERT_DIR="${REPO_ROOT}/../BuzzVirtual-Gen2-documentation/certificates"

Reply:
Oh yea that was missing all over the place, good catch.